src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Transition guide for OpenTelemetry cumulative sum metrics
+metaDescription: 'A transition guide about a change in how New Relic handles OpenTelemetry cumulative sum metrics, and how to update your queries and alerts for that change.'
+redirects: 
+---
+
+Our OTLP metric pipeline is being upgraded to better support how we handle [cumulative metrics](/docs/data-apis/understand-data/metric-data/cumulative-metrics). As part of this upgrade we're transitioning the underlying [metric data type](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types) for [monotonic cumulative sums](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#sums) from a `gauge` type to the `cumulativeCount` type. The `cumulativeCount` type allows you to query both the cumulative and delta value associated with reported data. 
+
+Starting March 14, 2023, new New Relic customers will use the new `cumulativeCount` type, but if you were a New Relic customer before that date, you won't have access to that metric type until June 13. On that date, the use of our `gauge` metric type for cumulative sums will be deprecated and reach end-of-life (EoL), and your cumulative sums will be stored as the `cumulativeCount` type. 
+
+If you were a customer using cumulative metrics before March 14, you should check and potentially modify existing queries and alerts that may be relying on those metrics. The sections below will teach you: 
+
+* How to know if your account is impacted
+* How to prepare for the EoL
+* How to preview the `cumulativeCount` functionality 
+
+<Callout variant="important">
+Non-monotonic cumulative sums will **not** be affected by this upgrade.
+</Callout>
+
+## Check your account for impact [#impact]
+
+If your account was sending OTLP cumulative sum metrics to New Relic before March 14, 2023, then that account will be impacted by the EoL on June 13, 2023. 
```

Comment:
Just to be safe, I'd add "monotonic": If your account was sending OTLP monotonic cumulative sum...."

Reply:
@tclayton-newr wanted to get your take on this. should we always be mentioning 'monotonic' when we reference this metric?

Reply:
Yes, while more verbose that is the most accurate statement of metrics that are affected by this EoL